### PR TITLE
Add ResNet152 and InceptionV4

### PR DESCRIPTION
**Pull request by jsalbert**
As @snormore suggested I added some new models: `DenseNet161`, `ResNet152`, `
Inception_v4`, `Inception_ResNet_v2`.

- `DenseNet161 is not loading well the pre-trained weights. 
- `Inception_ResNet_v2`is not yet available in the last public Keras version.

### CODE REVIEW

File: keras_model_specs/models/resnet152.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
+# ResNet-152 in Keras was posted by flyyufelix
+# https://gist.github.com/flyyufelix/7e2eafb149f72f4d38dd661882c554a6
+# without a license.
```
Comment (stephensolis) [inline keras_model_specs/models/resnet152.py]: It's also here: https://github.com/flyyufelix/cnn_finetune/blob/master/resnet_152.py (as MIT)

Comment (snormore) [reply keras_model_specs/models/resnet152.py]: Nice catch ✅

**Comment by snormore** [comment]
Finally 🍏  CI, I had to disable the VGG19 model loading in CI because it was OOMing.

**Comment by jsalbert** [comment]
Looks good to me @snormore

**Pull request merged**